ansible/roles/prometheus/templates/prometheus-ironic-exporter.json.j2

```diff
@@ -0,0 +1,18 @@
+{
+    "command": "gunicorn -b {{ api_interface_address }}:{{ prometheus_ironic_exporter_port }} --env IRONIC_CONFIG=/etc/prometheus/ironic.conf --access-logfile=/var/log/kolla/prometheus/ipe_access.log --error-logfile=/var/log/kolla/prometheus/ipe_errors.log ironic_prometheus_exporter.app.wsgi:application",
```

Comment:
Are there any other things using gunicorn? How are other WSGI servers handled in K-A? I thought it mostly used Apache.